README

```diff
@@ -223,19 +223,19 @@ Along with the developer mailing list archive:
     
 System Requirements
 ===================
-You need a platform that supports Sun Java SE 6.
```

Comment:
These instructions are for `Release Notes 3.2.1`
I would leave it as is ...

Reply:
Ok, I'll revert this changes.